docs/api-docs/getting-started/about-our-api.md

```diff
@@ -118,20 +114,20 @@ Server-to-Server request headers require Accept, X-Auth-Token, and Content-Type
 | `Transfer-Encoding` | `chunked` | Specifies the form of encoding used to transfer the resource. | `chunked`
 | `X-Rate-Limit-Requests-Left` | number | Details how many remaining requests your client can make in the current window before being rate-limited. In this case, you would expect to be able to make 6 more requests in the next 3000 milliseconds; on the 7th request within 3000 milliseconds, you would be rate-limited and would receive an HTTP 429 response. | `16101491` |
 | `X-Rate-Limit-Requests-Quota` | number | Shows how many API requests are allowed in the current window for your client. | `16101495` |
-| `X-Rate-Limit-Time-Reset-Ms`  | number | Shows how many milliseconds are remaining in the window. In this case, 3000 milliseconds – so, 3000 milliseconds after this request, the API quota will be refreshed. |`30000 `|
+| `X-Rate-Limit-Time-Reset-Ms` | number | Shows how many milliseconds are remaining in the window. In this case, 3000 milliseconds – so, 3000 milliseconds after this request, the API quota will be refreshed. |`30000 `|
 | `X-Rate-Limit-Time-Window-Ms` | number | Shows the size of your current rate-limiting window. | `9762` |
 
 ### Media types
 
-A media type is the format of the request or response body. The BigCommerce API accepts requests and responds in JSON. You should encode requests using the UTF-8 character set (other character sets might have unpredictable results).
+A media type is the format of the request or response body. BigCommerce APIs accept requests and send responses in JSON. You should encode requests using the UTF-8 character set. Other character sets may have unpredictable results.
 
 ### Content types
 
 ### Request content type
-When performing a request that contains a body (eg. POST or PUT), the type of content you are sending needs to be specified in the Content-Type header.
+When performing a request that contains a body (eg. POST or PUT), the type of content you are sending needs to be specified in the `Content-Type` header.

```

Comment:
Regarding i.e, and e.g --[style guide](https://intranet.bigcommerce.com/pages/viewpage.action?spaceKey=SST&title=Style+Guide+Word+List/) says we not use them but i'll make this your call.

Reply:
on the fence. I believe this was there before and that eg indicates enumerating existing cases -- so this should be the correct usage. I'll continue to reflect.